cioos_data_transform/cioos_data_transform/utils/odf.py

```diff
@@ -1,6 +1,169 @@
 import re
```

Comment:
I would move this file to the odf project folder since this isn't code that is shared amongst data transform projects

Reply:
could be project specific, I guess it depends on how we want to structure generally the cioos-data-transform repo and interpret the different "project" folders